src/chess/uciloop.h

```diff
@@ -76,6 +76,11 @@ class UciLoop {
                            const std::vector<std::string>& /*moves*/) {
     throw Exception("Not supported");
   }
+  // To support non-UCI commands this can be overridden in the engine class.
+  virtual bool CmdNonStandardUciCommand(const std::string& command) {
+	  throw Exception("Unknown command: " + command);
```

Comment:
I think default behaviour should be `return false;` rather than throwing exception?

Reply:
Done